### PR TITLE
Fix 0.116.0 compatibility

### LINKED ISSUE #43: Weather card is not working with 0.116.0b0
This is what the card looks like on version 0.115.x:
Screenshot_2020 09 30_23h41m19s_030_

And here is what the card looks like on version 0.116.0b0 (just gone):
Screenshot_2020 09 30_23h42m27s_031_

The HTML element is just empty:
Screenshot_2020 09 30_23h45m30s_032_

Let me know, if you need more info about this issue :)

### PR DESCRIPTION
**Pull request by kalkih**
Fixes compatibility issues with recent HA release 0.116.0

Resolves #43

### CODE REVIEW

**Comment by hmmbob** [comment]
Yes, 0.8.1 fixed it for me too.

File: src/main.js [hunk 1]
```diff
@@ -2,9 +2,8 @@ import WeatherEntity from './weather';
 import style from './style';
 import { handleClick } from './handleClick';
 
-const LitElement = window.LitElement || Object.getPrototypeOf(
-  customElements.get("hui-view"),
-);
+const LitElement = window.LitElement
+  || Object.getPrototypeOf(customElements.get("home-assistant") || customElements.get("hui-view"))
```
Comment (bramkragten) [inline src/main.js:6]: This won't work on Cast, use `hui-masonry-view` and as fallback `hui-view`

Comment (kalkih) [reply src/main.js:6]: Thanks Bram, appreciate it!

Comment (bramkragten) [reply src/main.js:6]: Another issue with this is, that the `home-assistant` element is not just a LitElement, it has a bunch of mixins you now also get in your card, that you should not get (like themes handling, causing issues in HA 2021.5).

Please fix this asap 😄

**Pull request merged**